en/view/acp_delete_file.php

```diff
@@ -0,0 +1,11 @@
+<?php
+	include_once "../../php/libs/database.php";
+    
+	date_default_timezone_set('Europe/Brussels');
+
+	$db = new Db();
+
+	$db->query("DELETE FROM `torrents` WHERE `id`=".$_GET['delete_id']."");

```

Comment:
What's the security on this? Can anyone delete any torrent file?

Reply:
I believe I had a simple function 'valid_user' in upload.php to see if the logged in user has a valid session key. It's at least some security, I didn't know much about this when I originally made this project.

Reply:
Only users with 'uploaderstatus'=99 will be able to delete torrents from the acp menu.

For anyone other than an admin; torrents will not appear on acp as they can delete their own torrents from my-torrents.php

Reply:
Ahh yes I see; adding that would make things a little bit more secure.
Would you be able to suggest edit those things for me? I'm learning PHP as I go with this project.

Reply:
@AzukaChan I'll check it out during the week, busy schedule. I'll see which kind of security we can set on this API. If you want and you use Discord, you can always mail me your tag#numbers to contact@kevindurant.be.